tests/core/symbolutil.c

```diff
@@ -13,12 +13,14 @@
     TEST_ENTRY_SIMPLE ("Core/SymbolUtil", test_symbolutil, NAME)
 
 TEST_LIST_BEGIN (symbolutil)
+#ifndef HAVE_QNX
```

Comment:
It would be better to disable the test-suite here:
https://github.com/frida/frida-gum/blob/master/tests/gumtest.c#L100